src/makie.jl

```diff
@@ -526,4 +705,69 @@ function first_animation()
             recordframe!(io)  # record a new frame
         end
     end
+end
+
+function grid_spheres_and_rectangle_as_plate()    
+    Random.seed!(123)
+    rectMesh = FRect3D(Vec3f0(-1,-1,2.1), Vec3f0(22,11,0.5))
+    recmesh = GeometryBasics.mesh(rectMesh)
+    colors = [RGBA(rand(4)...) for v in recmesh.position]
```

Comment:
This could be something like `repeat(RGBA(rand(4)...), length(recmesh.position))`.